src/common/utility_win.cpp

```diff
@@ -29,10 +29,29 @@
 
 #include <QLibrary>
 
+#include "common/result.h"
+#include "common/winrthelper/winrthelper.h"
+
 extern Q_CORE_EXPORT int qt_ntfs_permission_lookup;
 
-static const char systemRunPathC[] = "HKEY_LOCAL_MACHINE\\Software\\Microsoft\\Windows\\CurrentVersion\\Run";
-static const char runPathC[] = "HKEY_CURRENT_USER\\Software\\Microsoft\\Windows\\CurrentVersion\\Run";
+namespace {
+const char systemRunPathC[] = "HKEY_LOCAL_MACHINE\\Software\\Microsoft\\Windows\\CurrentVersion\\Run";
+const char runPathC[] = "HKEY_CURRENT_USER\\Software\\Microsoft\\Windows\\CurrentVersion\\Run";
+const QString rtHelperNameC()
+{
+    return QStringLiteral(APPLICATION_EXECUTABLE "_winrt");
+}
```

Comment:
could be a const char[] just like the defines above, no?

Reply:
the first parameter from QLibaray takes a QString not char array, anyhow I will pass a absolute string to ensure we are loading the right lib...